R/validate_inputs.R

```diff
@@ -115,7 +115,6 @@ do_validate_population <- function(population) {
 do_validate_programme <- function(programme, shape, strict = TRUE) {
   assert_file_extension(programme, "csv")
   data <- read_csv(programme$path, header = TRUE)
-  data$art_current <- as.numeric(data$art_current)
```

Comment:
This looks like it might want shifting down the function rather than removing (i.e., after line 122). Was this previously put in to cope with things like an all-missing columns coming through as a logical NA?

Reply:
Code originally added in https://github.com/mrc-ide/hintr/commit/c2a4a863378d2778e9be6ebb344d6c342423aee3 if that helps

Reply:
Oh yeah, this is duped up. It is still on line 133 which is just before we validate the contents of that column